io/pci/distro_tools.py

```diff
@@ -87,6 +148,21 @@ def test(self):
         elif self.tool == "lsprop":
             for val in self.option:
                 self.lsprop(val)
+        elif self.tool == "lsvio":
+            for val in self.option:
```

Comment:
do we need for loop for `self.option` ?

Reply:
here split() is their that's why it needed, with out that also it worked.i will change that.